charts/kubearchive/templates/api_server/api_server.yaml

```diff
@@ -24,31 +24,14 @@ spec:
             - name: tls-secret
               readOnly: true
               mountPath: /etc/kubearchive/ssl/
+          envFrom:
+            - secretRef:
+                name: {{ .Values.database.secret }}
           env:
             - name: GIN_MODE
               value: {{ ternary "release" "debug" .Values.global.production }}
-            - name: POSTGRES_DB
-              valueFrom:
-                secretKeyRef:
-                  name: {{ tpl .Values.database.secret . }}
-                  key: POSTGRES_DB
-            - name: POSTGRES_USER
-              valueFrom:
-                secretKeyRef:
-                  name: {{ tpl .Values.database.secret . }}
-                  key: POSTGRES_USER
-            - name: POSTGRES_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: {{ tpl .Values.database.secret . }}
-                  key: POSTGRES_PASSWORD
-            - name: POSTGRES_URL
-              valueFrom:
-                secretKeyRef:
-                  name: {{ tpl .Values.database.secret . }}
-                  key: POSTGRES_URL
             - name: POSTGRES_PORT
-              value: "{{ required "A Postgres port must be specified" .Values.database.service.port }}"
+              value: "5432"
```

Comment:
Why is the postgres port needed here?

Reply:
The same as the sink, in [this function](https://github.com/maruiz93/kubearchive/blob/main/pkg/database/connection.go#L36) we establish the postgres db connection string which needs the port. The port is one of the variables and is injected (like the rest) from the env variable.